lib/ProductOpener/Display.pm

```diff
@@ -264,6 +273,21 @@ sub init()
 		return 301;
 	}
 	
+	
+	# Allow cc and lc overrides as query parameters
```

Comment:
I guess this works, but we probably need to add some docs that ie. https://de.openfoodfacts.org/produkt/123?lc=fr will generate links in the website like https://fr.openfoodfacts.org/produit/123, because that could be unexpected.